updater/spec/dependabot/environment_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "dependabot/environment"
+
+RSpec.describe Dependabot::Environment do
+  subject(:environment) { described_class }
+
+  describe "::debug_enabled?" do
+    after do
+      # Reset the memoisation after each test
+      environment.remove_instance_variable(:@debug_enabled)
+    end
+
+    it "is false by default" do
+      allow(environment).to receive(:job_definition).and_return({})
+      allow(ENV).to receive(:fetch).with("DEPENDABOT_DEBUG", false).and_return(false)
+
+      expect(environment).not_to be_debug_enabled
```

Comment:
where is `be_debug_enabled` defined?

Reply:
Ah, this is an RSpec-isim, `be_debug_enabled` is translated to `environment.debug_enabled? to be true` under the hood.